sdk/search/azure-search-documents/src/test/java/com/azure/search/documents/test/environment/models/HotelSearchException.java

```diff
@@ -0,0 +1,33 @@
+package com.azure.search.documents.test.environment.models;
```

Comment:
Might be good to have the cycle a level deeper just to make sure the recursive stack push/check is working correctly (and continues to do so when changed).